source/upgrade.txt

```diff
@@ -132,6 +132,12 @@ Version 4.0 Breaking Changes
   UUID values must explicitly specify which representation to use. You can
   specify the representation in the ``uuidRepresentation`` property of
   ``MongoClientSettings``.
+
+  The UUID representation that you specify strictly controls how the driver
+  decodes UUIDs. For example, if you specify any of the ``LEGACY`` UUID
+  representations, the driver decodes only binary objects of subtype 3
```

Comment:
Suggestion:
Link to the API documentation of UuidRepresentation enum since it may not be clear what the various LEGACY UUID representations are. For example, here's the API docs for v4.0 of the driver:

http://mongodb.github.io/mongo-java-driver/4.0/apidocs/bson/org/bson/UuidRepresentation.html

Alternatively, if the intention is to present a concrete example, use the JAVA_LEGACY representation as the example.